src/App.jsx

```diff
@@ -1,104 +1,32 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
+
+function findUserId(UserId) {
```

Comment:
```suggestion
function findUserById(userId) {
```
- You lock for a user by id, not for the user's id
- Arguments should be in camelCase

Reply:
Not fixed